packages/hot-updater/src/utils/createZip.ts

```diff
@@ -18,6 +18,11 @@ export const createZip = async ({
     files.sort();
 
     for (const file of files) {
+      if (file.endsWith(".map")) {
+        // skip sourcemap files
+        continue;
+      }
```

Comment:
Since createZip is a utility that only compresses zip files, how about controlling its behavior using options?

```js
createZip({
   outfile: "..",
   targetDir: "..",
   excludeExts: [".map"]
});
```

Reply:
done!